src/main/java/randoop/condition/OutcomeTable.java

```diff
@@ -0,0 +1,109 @@
+package randoop.condition;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import randoop.test.ExpectedExceptionGenerator;
+import randoop.test.ExtendGenerator;
+import randoop.test.InvalidCheckGenerator;
+import randoop.test.PostConditionCheckGenerator;
+import randoop.test.TestCheckGenerator;
+
+/** Records the outcome of checking all of the conditions for a method. */
+public class OutcomeTable {
+
+  /** Indicates whether this table is empty. */
+  private boolean isEmpty = true;
+
+  /** Indicates whether this table has a valid entry. */
```

Comment:
What is the definition of "valid"?  Uses later in the class suggest that this class does not use a standard definition, which is confusing (and especially confusing when not documented, but it's better to use a standard definition *and* document it).